games-strategy/freeciv/freeciv-3.0.0.ebuild

```diff
@@ -170,9 +170,13 @@ src_install() {
 			# delete freeciv-manual from the GAMES_BINDIR, because it's useless.
 			# Note: to have it localized, it should be ran from _postinst, or
 			# something like that, but then it's a PITA to avoid orphan files...
-			./tools/freeciv-manual || die
-			docinto html
-			dodoc classic*.html
+			# freeciv-manual only supports one ruleset argument at a time.
+			for RULESET in alien civ1 civ2 civ2civ3 classic experimental multiplayer sandbox
+			do
+				./tools/freeciv-manual -r ${RULESET} || die
+				docinto rulesets/${RULESET}
```

Comment:
The game works fine for me. My only concern is with the path for these rulesets. It is conventional to put HTML files in the `html` subdirectory, so either make this `docinto html/${RULESET}` or `docinto html/rulesets/${RULESET}`. I'm assuming the game itself doesn't use these files.

Reply:
Ack. I'll update the ebuild!